bin/hdfcoinc/pycbc_page_snrratehist

```diff
@@ -104,6 +102,29 @@ if fstat is not None and not args.closed_box:
                    xerr=args.bin_size/2, label='Foreground',
                    mec='none', fmt='o', ms=1, capthick=0, elinewidth=4,  color='#ff6600')
 
+    # We need to plot a second histogram "errorbar" with the hierarchically
```

Comment:
The plotting of hierarchically removed foreground triggers should remove those triggers from being plotted in orange, instead of plotting on top of them in red. If say there are 4 foreground triggers that are louder than the background in a snr-bin on the plot and the hierarchical removal code has been run with `--max-hier-removal` too low, there is a potential to see 4 orange foreground triggers in that bin, but 3 or fewer red squares. This may lead the viewer of these plots to see 7 (!) triggers on the plot, instead of say 1 (or greater) orange foreground triggers and 3 (or fewer) red hierarchically removed foreground triggers. This shouldn't be hard to patch. But I'll have to do it later.